en/components/grids_templates/column_hiding.md

```diff
@@ -437,6 +437,176 @@ If all went well, this is how our column hiding UI component should look like:
+
+ >[!NOTE]
+ >If the component is using an [`Emulated`](../themes/component-themes.md#view-encapsulation) ViewEncapsulation, it is necessary to `penetrate` this encapsulation using `::ng-deep`:
+
+```scss
+:host {
+    ::ng-deep {
+        @include igx-column-hiding($custom-column-hiding-theme);
+        .igx-column-hiding {
+            @include igx-button($custom-button);
+        }
+    }
+}
+```
+
+#### Defining a color palette
+
+Instead of hardcoding the color values like we just did, we can achieve greater flexibility in terms of colors by using the [`igx-palette`]({environment:sassApiUrl}/index.html#function-igx-palette) and [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) functions.
+
+`igx-palette` generates a color palette based on the primary and secondary colors that are passed:
+
+```scss
+$yellow-color: gold;
+$blue-color: steelblue;
+
+$custom-palette: igx-palette($primary: $blue-color, $secondary: $yellow-color);
+```
+
+And then with [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) we can easily retrieve color from the palette. 
+
+```scss
+$custom-column-hiding-theme: igx-column-hiding-theme(
+    $palette: $custom-palette,
+    $title-color: igx-color($custom-palette, "secondary", 400),
+    $background-color: igx-color($custom-palette, "primary", 200)
+);
+
+$custom-button: igx-button-theme(
+    $palette: $custom-palette,
+    $flat-text-color: igx-color($custom-palette, "secondary", 400),
+    $disabled-color: black
+);
+```
+
+>[!NOTE]
+>The `igx-color` and `igx-palette` are powerful functions for generating and retrieving colors. Please refer to [`Palettes`](../themes/palette.md) topic for detailed guidance on how to use them.
+
+#### Using Schemas
+
+Going further with the theming engine, you can build a robust and flexible structure that benefits from [**schemas**](../themes/schemas.md). A **schema** is a recipe of a theme.
+
+```scss
+// Extending the dark column hiding schema
+$custom-column-hiding-schema: extend($_dark-column-hiding,
+    (
+        title-color:(
+            igx-color: ("secondary", 400)
+        ),
+        background-color:(
+            igx-color: ("primary", 200)
+        )
+    )
+);
+// Extending the dark button schema
+$custom-button-schema: extend($_dark-button,
+    (           
+        flat-text-color:(
+            igx-color:("secondary", 500)
+        ),
+        disabled-color:(
+            igx-color:("primary", 700)
+        )
+    )
+);
+```
+
+In order to apply our custom schemas we have to **extend** one of the globals ([`light`]({environment:sassApiUrl}/index.html#variable-light-schema) or [`dark`]({environment:sassApiUrl}/index.html#variable-dark-schema)), which is basically pointing out the components with a custom schema, and after that add it to the respective component themes:
+
+```scss
+// Extending the global dark-schema
+$custom-dark-schema: extend($dark-schema,(
+    igx-grid-column-hiding: $custom-column-hiding-schema,
```

Comment:
The schema property for the grid column hiding is `igx-column-hiding`:
[https://staging.infragistics.com/products/ignite-ui-angular/docs/sass/latest/index.html#variable-dark-schema](https://staging.infragistics.com/products/ignite-ui-angular/docs/sass/latest/index.html#variable-dark-schema)